code/sound/audiostr.h

```diff
@@ -66,6 +66,9 @@ void audiostream_set_volume(int i, float volume);
 // see if a particular stream is paused
 int audiostream_is_paused(int i);
 
+// get duration of a stream in milliseconds
```

Comment:
This should now be changed to seconds
```suggestion
// get duration of a stream in seconds
```